packages/e2e-tests/utils/account.js

```diff
@@ -1,99 +1,77 @@
 const {
-    connect,
-    keyStores: { InMemoryKeyStore },
     utils: {
-        KeyPair,
         format: { parseNearAmount },
     },
 } = require("near-api-js");
-const { parseSeedPhrase } = require("near-seed-phrase");
+const BN = require("bn.js");
 
-const { walletNetwork } = require("./config");
+const nearApiJsConnection = require("./connectionSingleton");
+const { getKeyPairFromSeedPhrase } = require("./helpers");
 
-const getDefaultConfig = () => ({
-    networkId: walletNetwork,
-    nodeUrl: process.env.NODE_URL || "https://rpc.testnet.near.org",
-    walletUrl: process.env.WALLET_URL || "https://wallet.testnet.near.org",
-    keyStore: new InMemoryKeyStore(),
-});
-
-function getKeyPairFromSeedPhrase(seedPhrase) {
-    return KeyPair.fromString(parseSeedPhrase(seedPhrase).secretKey);
+class E2eTestAccount {
+    constructor(accountId, seedPhrase, parentNearApiJsAccount) {
+        this.accountId = accountId;
+        this.seedPhrase = seedPhrase;
+        this.parentNearApiJsAccount = parentNearApiJsAccount;
+    }
+    async initialize() {
```

Comment:
Feels like initializing a new account would be:
1. ensure we have a nearAPIJS instance
2. create the account
3. Set the key in the keystore only if the account creation was a success

Is there any benefit we get from separating out the actual creation of the account to a separate function call than `initialize()`?  Seems like it just adds an extra call to ever place someone makes one of these objects?